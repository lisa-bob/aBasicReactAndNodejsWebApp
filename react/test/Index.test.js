import React from "react"
import Index from "../src/js/pages/Index"

test('Render Index page', () => {
    const wrapper = shallow(
        <Index />
    );

    expect(wrapper).toMatchSnapshot();
});