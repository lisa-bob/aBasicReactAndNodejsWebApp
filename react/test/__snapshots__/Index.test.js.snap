// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Index page 1`] = `
<div>
  <h2
    className="red-color"
  >
    Index Page
  </h2>
</div>
`;
